test/Microsoft.ML.TestFrameworkCommon/Utility/LibraryLoader.cs

```diff
@@ -0,0 +1,219 @@
+﻿// Taken from https://github.com/mellinoe/nativelibraryloader/blob/586f9738ff12688df8f0662027da8c319aee3841/NativeLibraryLoader/LibraryLoader.cs#L182-L216https://github.com/mellinoe/nativelibraryloader/blob/586f9738ff12688df8f0662027da8c319aee3841/NativeLibraryLoader/LibraryLoader.cs
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.ML.TestFrameworkCommon.Utility
+{
+    /// <summary>
+    /// Exposes functionality for loading native libraries and function pointers.
+    /// </summary>
+    public abstract class LibraryLoader
+    {
+        /// <summary>
+        /// Loads a native library by name and returns an operating system handle to it.
+        /// </summary>
+        /// <param name="name">The name of the library to open.</param>
+        /// <returns>The operating system handle for the shared library.</returns>
+        public IntPtr LoadNativeLibrary(string name)
```

Comment:
>LoadNativeLibrary [](http://example.com/codeflow?start=22&length=17)

nit: use => syntax?

Reply:
I took the code from the github link mentioned on top and just left it as is. I can switch them all though.

Reply:
fine with me either way. I was just suggesting the style which we usually use.